extensions/cornerstone/src/init.js

```diff
@@ -275,3 +287,228 @@ const _connectToolsToMeasurementService = measurementService => {
     }
   );
 };
+
+const _enableReferenceLines = () => {
+  const waitForImageRendered = enabledElement =>
+    new Promise(resolve => {
+      const onImageRenderedCallback = () => {
+        enabledElement.removeEventListener(
+          cornerstone.EVENTS.IMAGE_RENDERED,
+          onImageRenderedCallback
+        );
+        resolve();
+      };
+      enabledElement.addEventListener(
+        cornerstone.EVENTS.IMAGE_RENDERED,
+        onImageRenderedCallback
+      );
+    });
+
+  const renderReferenceLines = ({ detail: { enabledElement } }) => {
+    const { activeViewportIndex } = window.store.getState().viewports;
```

Comment:
This is the last `window.store` reference. I think we might _have_ to keep it for now, but we should create a follow-up issue where we find a better way to provide this information to extensions.

Reply:
Yes I agree.